src/Azure.Functions.Cli/Actions/InternalUseAction.cs

```diff
@@ -54,12 +53,6 @@ public override Task RunAsync()
             {
                 switch (action)
                 {
-                    case InternalAction.SetupUrlAcl:
```

Comment:
This switch should go since the other actions have been removed. Left here since we'll need to iterate on SSL (haven't tested that as part of the port)